src/Tribe/Assets.php

```diff
@@ -154,13 +174,17 @@ public function enqueue( $forcibly_enqueue = null ) {
 			} else {
 				wp_enqueue_style( $asset->slug );
 			}
+
+			$this->assets[ $key ]->already_enqueued = true;
```

Comment:
This method gets called many times, each time the `$this->assets` will be passed to the loop. In order for a change to one of the assets to persist it looks like I would need to make `$asset` a reference for that to work. Do you think that's preferable? It would be more concise, but sometimes people forget they're working with a reference. I could go either way on this one.

http://php.net/manual/en/control-structures.foreach.php